jsapp/js/components/modalForms/bulkEditSubmissionsForm.es6

```diff
@@ -212,6 +212,12 @@ class BulkEditSubmissionsForm extends React.Component {
     }
   }
 
+  disableEdit(questionType) {
```

Comment:
Since this returns a boolean, the name of the function should get changed to match our [coding conventions](https://kobotoolbox.github.io/articles/kpi-frontend-development) - something like `isEditDisabled`.